_config/routes.yml

```diff
@@ -2,9 +2,10 @@
 Name: commentroutes
 After: framework/routes#coreroutes
 ---
-Director:
+SilverStripe\Control\Director:
   rules:
+    comments: SilverStripe\Comments\Controllers\CommentingController
     # handle old 2.4 style urls
-    'CommentingController//$Action/$ID/$OtherID': 'CommentingController'
-    'PageComments/$Action/$ID/$OtherID': 'CommentingController'
-    'PageComments_Controller/$Action/$ID/$OtherID': 'CommentingController'
+    CommentingController//$Action/$ID/$OtherID: SilverStripe\Comments\Controllers\CommentingController
```

Comment:
This is a duplicate route to the `comments` above. I suggest moving this to a unified route.